diff --git a/node_modules/ddddocr-node/dist/index.js b/node_modules/ddddocr-node/dist/index.js
index 5ade8cc..cca1d6e 100644
--- a/node_modules/ddddocr-node/dist/index.js
+++ b/node_modules/ddddocr-node/dist/index.js
@@ -41,12 +41,11 @@ class DdddOcr {
      */
     constructor(logSeverityLevel = 4) {
         this._ocrMode = MODEL_TYPE.OCR;
-        const root = node_path_1.default.join(__dirname, '..');
-        this._ocrOnnxPath = `${root}/onnx/common_old.onnx`;
-        this._charsetPath = `${root}/onnx/common_old.json`;
-        this._ocrBetaOnnxPath = `${root}/onnx/common.onnx`;
-        this._charsetBetaPath = `${root}/onnx/common.json`;
-        this._ocrDetectionOnnxPath = `${root}/onnx/common_det.onnx`;
+        this._ocrOnnxPath = 'common_old.onnx';
+        this._charsetPath = 'common_old.json';
+        this._ocrBetaOnnxPath = 'common.onnx';
+        this._charsetBetaPath = 'common.json';
+        this._ocrDetectionOnnxPath = 'common_det.onnx';
         this._ocr = new Ocr_1.OCR(this._ocrOnnxPath, this._charsetPath).setLogSeverityLevel(logSeverityLevel);
         this._ocrBeta = new Ocr_1.OCR(this._ocrBetaOnnxPath, this._charsetBetaPath).setLogSeverityLevel(logSeverityLevel);
         this._detection = new Detection_1.Detection(this._ocrDetectionOnnxPath).setLogSeverityLevel(logSeverityLevel);
diff --git a/node_modules/ddddocr-node/src/index.ts b/node_modules/ddddocr-node/src/index.ts
index 18e9b2d..a666dd6 100644
--- a/node_modules/ddddocr-node/src/index.ts
+++ b/node_modules/ddddocr-node/src/index.ts
@@ -42,15 +42,13 @@ class DdddOcr {
      * @param logSeverityLevel Log severity level. See https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/common/logging/severity.h
      */
     constructor(logSeverityLevel: LogSeverityLevel = 4) {
-        const root = path.join(__dirname, '..');
+        this._ocrOnnxPath = 'common_old.onnx';
+        this._charsetPath = 'common_old.json';
 
-        this._ocrOnnxPath = `${root}/onnx/common_old.onnx`;
-        this._charsetPath = `${root}/onnx/common_old.json`;
+        this._ocrBetaOnnxPath = 'common.onnx';
+        this._charsetBetaPath = 'common.json';
 
-        this._ocrBetaOnnxPath = `${root}/onnx/common.onnx`;
-        this._charsetBetaPath = `${root}/onnx/common.json`;
-
-        this._ocrDetectionOnnxPath = `${root}/onnx/common_det.onnx`;
+        this._ocrDetectionOnnxPath = 'common_det.onnx';
 
         this._ocr = new OCR(this._ocrOnnxPath, this._charsetPath).setLogSeverityLevel(logSeverityLevel);
         this._ocrBeta = new OCR(this._ocrBetaOnnxPath, this._charsetBetaPath).setLogSeverityLevel(logSeverityLevel);
